sdk-extensions/jaeger-remote-sampler/src/test/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerProviderTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.trace.jaeger.sampler;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigurableSamplerProvider;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.ServiceLoader;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class JaegerRemoteSamplerProviderTest {
+
+  @Test
+  void serviceProvider() {
+    ServiceLoader<ConfigurableSamplerProvider> samplerProviders =
+        ServiceLoader.load(ConfigurableSamplerProvider.class);
+    Iterator<ConfigurableSamplerProvider> iterator = samplerProviders.iterator();
+    iterator.forEachRemaining(
+        configurableSamplerProvider ->
+            assertThat(configurableSamplerProvider)
+                .isInstanceOf(JaegerRemoteSamplerProvider.class));
```

Comment:
```suggestion
    assertThat(samplerProviders).hasAtLeastOneElementOfType(JaegerRemoteSamplerProvider.class);
```

Found this while writing a similar test :)

Reply:
Or you can check the name too with this pattern

https://github.com/open-telemetry/opentelemetry-java-contrib/pull/64/files#diff-e9cb51992b77bc5666278183358a03a9c3e5b0ec6a500277e9e5681086d3f8b4R20